packages/litqa/tests/test_litqa_env.py

```diff
@@ -0,0 +1,56 @@
+from typing import cast
```

Comment:
Some of the `test_task` stuff in PQA needs to come here too, and also the stub data